### PR TITLE
docker load support.

### PR DESCRIPTION
**Pull request by Katsuya-Tomioka**
Signed-off-by: Katsuya Tomioka <katsuya@basistech.com>

### CODE REVIEW

**Comment by rhuss** [comment]
Thanks a lot ! I will have review next week.

File: src/main/java/io/fabric8/maven/docker/config/ImageConfiguration.java [hunk 1]
```diff
@@ -33,6 +33,9 @@
     @Parameter
     private String registry;
 
+    @Parameter
+    private String file;
```
Comment (rhuss) [inline src/main/java/io/fabric8/maven/docker/config/ImageConfiguration.java]: Actually I would move this to the `<build>` configuration as it is about *creating images*. There is should be probably called something like `dockerArchive` on the same footprint as `dockerFile` or `dockerFileDir`. Also there should be a check, that only one of `dockerArchive` and `dockerFile` is given.

**Comment by rhuss** [comment]
Ah, yes, and when adding a new build option don't forget to update `PropertyConfigHandler`, too (along with a minor doc update)

**Comment by Katsuya-Tomioka** [comment]
I moved dockerFile part to the build side. Also added a bunch of tests to pass the coverage check. Hope you don't mind. If you could review/edit the doc, that'd be great.

**Comment by rhuss** [comment]
[merge]

**Pull request merged**